packages/frontend/src/features/dashboardFiltersV2/ActiveFilters/Filter.module.css

```diff
@@ -20,6 +20,10 @@
     text-overflow: ellipsis;
 }
 
+.hasBadge {
+    padding-right: 6px !important;
+}
```

Comment:
[np]: can this be a `pr={6}` on the `Button` directly? or this is a preferred for non-theme values in v8?

Reply:
done